ps_healthcheck.php

```diff
@@ -0,0 +1,138 @@
+<?php
+/**
+ * 2007-2020 PrestaShop SA and Contributors.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors.
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+if (!defined('_CAN_LOAD_FILES_')) {
+    exit(1);
+}
+
+if (file_exists(__DIR__ . '/vendor/autoload.php')) {
+    require_once __DIR__ . '/vendor/autoload.php';
+}
+
+use PrestaShop\Module\HealthCheck\Repository\HealthCheckConfigRepository;
+use PrestaShop\PrestaShop\Adapter\SymfonyContainer;
+
+/**
+ * Class Ps_Healthcheck.
+ */
+class Ps_Healthcheck extends Module
+{
+    const MODULE_NAME = 'ps_healthcheck';
+    /**
+     * @var HealthCheckConfigRepository
+     */
+    private $repository;
+
+    protected function loadDefaultSettings()
+    {
+        $this->name = static::MODULE_NAME;
+        $this->author = 'PrestaShop';
+        $this->version = '0.1.0';
```

Comment:
These data should came from the Manifest.php file to prevent duplicate

Reply:
@eternoendless didnt agree to use the manifest file to store the module attributes for now because it duplicate the config.xml infos

I changed it. these infos stay in the __construct and only the bundles are declared in the manifest.php file